timesketch/lib/analyzers/domain.py

```diff
@@ -66,22 +67,37 @@ def run(self):
             tld = '.'.join(domain.split('.')[-2:])
             tld_counter[tld] += 1
 
+        domain_count_array = numpy.array(list(domain_counter.values()))
+        domain_20th_percentile = int(numpy.percentile(domain_count_array, 20))
+        domain_85th_percentile = int(numpy.percentile(domain_count_array, 85))
+
+        top_domains = [
+            x for x, y in domain_counter.most_common()
+                if y >= domain_85th_percentile]
+        rare_domains = [
+            x for x, y in domain_counter.most_common()
+                if y <= domain_20th_percentile]
+
         satellite_emoji = emojis.get_emoji('SATELLITE')
         for domain, count in iter(domain_counter.items()):
             emojis_to_add = [satellite_emoji]
             tags_to_add = []
-            text = '{0:s} seen {1:d} times'.format(domain, count)
 
             cdn_provider = utils.get_cdn_provider(domain)
             if cdn_provider:
                 tags_to_add.append('known-cdn')
                 cdn_counter[cdn_provider] += 1
 
+            if domain in top_domains:
+                tags_to_add.append('top_domain')
```

Comment:
Rename to "common_domain" as "common" is a better antonym for "rare" than "top" is.

Reply:
done